cluster/ubuntu/util.sh

```diff
@@ -294,20 +294,21 @@ function detect-nodes {
   done
 
   if [[ -z "${KUBE_NODE_IP_ADDRESSES[@]}" ]]; then
-    echo "Could not detect Kubernetes node nodes. Make sure you've launched a cluster with 'kube-up.sh'" >&2
+    echo "Could not detect Kubernetes node nodes.\
+    Make sure you've launched a cluster with 'kube-up.sh'" >&2
     exit 1
   fi
 }
 
 # Instantiate a kubernetes cluster on ubuntu
 function kube-up() {
-  source "${KUBE_ROOT}/cluster/ubuntu/${KUBE_CONFIG_FILE-"config-default.sh"}"
+  source "${KUBE_ROOT}/cluster/ubuntu/${KUBE_CONFIG_FILE:-config-default.sh}"
 
-  # ensure the binaries are well prepared
-  if [ ! -f "ubuntu/binaries/master/kube-apiserver" ]; then
-    echo "No local binaries for kube-up, downloading... "
-    "${KUBE_ROOT}/cluster/ubuntu/build.sh"
+  # downloading tarball release
+  if [[ -d "${KUBE_ROOT}/cluster/ubuntu/binaries" ]]; then
```

Comment:
This is a logic change, and I do not understand the system well enough to say whether it is good or bad.